podcasts/episodes.go

```diff
@@ -39,22 +39,23 @@ func (podcast *Podcast) Episodes(index int, count int) []*Episode {
 	for i := 0; i < pagesToFetch; i++ {
 		go func(index int) {
 			fetchedEpisodes, err := podcast.scrapEpisodesByPodcast(fmt.Sprintf("%s/-episodes?page=%d", podcast.URL, startPage+index))
-			if err != nil {
+			if err == nil {
+				episodesChan <- fetchedEpisodes
+			} else {
 				log.Println(err)
 			}
```

Comment:
the problem is if there is an error there will be a deadlock at line 52

Reply:
Didn't realize that. Will revert this change.